src/ui/src/data/matchReview.ts

```diff
@@ -17,3 +17,41 @@ export async function getRecordMatch(id: string | null): Promise<RecordMatch> {
     );
   }
 }
+
+export async function linkRecordAndMatch(
+  id: string | null,
+): Promise<RecordMatch> {
+  const response = await fetch(`${API_URL}/demo/record/${id}/link`, {
+    method: "POST",
+  });
```

Comment:
```suggestion
    method: "POST",
     credentials: "include",
  });
```

Reply:
I'm guessing this is necessary because in dev we are using 2 hosts, and the default samesite value doesn't work.  I do wonder about this though, as I see were setting the cookie samesite value to "lax".  But if it works in dev, then I guess lets just go with it, it shouldn't be an issue at all in demo site since everything will come from recordlinker.dibbs.demo

> Note that if a cookie's [SameSite](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Set-Cookie#samesitesamesite-value) attribute is set to Strict or Lax, then the cookie will not be sent cross-site, even if credentials is set to include.

Reply:
I was going to make the recommendation that we think about moving to use React Query (especially Mutation) to help us handle these fetches. I'm less familiar with Next JS than Johanna but it seems to handle credentials (among other things that we'd want handled automatically).